docs/Reference/API-Objects.md

```diff
@@ -93,17 +93,21 @@ Returned by [`txpool_besuPendingTransactions`](API-Methods.md#txpool_besupending
 
 | Key | Type | Value |
 |-----|:----:|-------|
-| **from**             | Data, 20&nbsp;bytes | Address of the sender.                                              |
-| **gas**              | Quantity            | Gas provided by the sender.                                         |
-| **gasPrice**         | Quantity            | Gas price, in wei, provided by the sender.                          |
-| **hash**             | Data, 32&nbsp;bytes | Hash of the transaction.                                            |
-| **input**            | Data                | Data sent with the transaction to create or invoke a contract.      |
-| **nonce**            | Quantity            | Number of transactions made by the sender before this one.          |
-| **to**               | Data, 20&nbsp;bytes | Address of the receiver. `null` if a contract creation transaction. |
-| **value**            | Quantity            | Value transferred, in wei.                                          |
-| **v**                | Quantity            | ECDSA Recovery ID.                                                  |
-| **r**                | Data, 32&nbsp;bytes | ECDSA signature r.                                                  |
-| **s**                | Data, 32&nbsp;bytes | ECDSA signature s.                                                  |
+| **accessList**           | Array               | List of addresses and storage keys the transaction plans to access. Used in [`ACCESS_LIST` transactions](../Concepts/Transactions/Transaction-Types.md#access_list-transactions) and may be used in [`EIP1559` transactions](../Concepts/Transactions/Transaction-Types.md#eip1559-transactions). |
```

Comment:
do we have a way to mark these as nullable or optional?  same q for gasPrice, maxFeePerGas, maxPriorityFeePerGas